pyproject.toml

```diff
@@ -33,6 +32,9 @@ dev = [
     "pytest",
     "ruff==0.9.3"
 ]
+tensorboard = [
+    "tbparse==0.0.9"
```

Comment:
pin because this package is not super heavily maintained and we don't want want a downstream breakage to break `trackio`